apps/chat/src/components/Marketplace/ToolsetsDetails/ToolsetDetails.tsx

```diff
@@ -55,11 +57,19 @@ export function ToolsetDetails({
       (e) =>
         getGroupMarketplaceEntityKey(entity) ===
           getGroupMarketplaceEntityKey(e) &&
-        (!isMyWorkspaceTab || installedToolsetsIds.has(e.reference)),
+        (!isMyWorkspaceTab ||
+          installedToolsetsIds.has(e.reference) ||
+          isSuggested),
```

Comment:
boolean check faster and cheaper than O(1) search in Set
```suggestion
        (!isMyWorkspaceTab || isSuggested ||
          installedToolsetsIds.has(e.reference)),
```